service/src/main/java/bio/terra/workspace/app/controller/ControlledAwsResourceApiController.java

```diff
@@ -208,14 +278,14 @@ public ResponseEntity<ApiCreatedControlledAwsS3StorageFolder> createAwsS3Storage
   @Traced
   @Override
   public ResponseEntity<ApiAwsS3StorageFolderResource> getAwsS3StorageFolder(
-      UUID workspaceUuid, UUID resourceId) {
+      UUID workspaceUuid, UUID resourceUuid) {
```

Comment:
In most of the codebase I _think_ we use `workspaceId` and `resourceId` more than `_Uuid`. If the goal here is to standardize on one I'd prefer `_Id`, but I'm also fine using both versions.
Open to more input on that one, though

Reply:
This was ONLY to keep the naming consistent. 
in OpenApi `*id` is used
In implementation code `*uuid` was used more than `*id` (by overall count), hence this change.